packages/lodestar/src/sync/interface.ts

```diff
@@ -58,14 +59,29 @@ export interface ISyncModules {
   wsCheckpoint?: phase0.Checkpoint;
 }
 
+export enum PendingBlockType {
+  UNKNOWN_BLOCK = "UNKNOWN_BLOCK",
+  UNKNOWN_PARENT = "UNKNOWN_PARENT",
```

Comment:
If used in metrics, don't use all caps for the value as it's not the best for reading. I would use
```
= "unknown_block"
```
or
```
 = "unknownBlock"
```